src/components/data-processing/StepsIndicator.jsx

```diff
@@ -8,6 +8,12 @@ const StepsIndicator = (props) => {
     currentStep: '#ff6f00', // orange
     completed: '#009930', // blue
   };
+
+  const testClasses = {
```

Comment:
I would just put 'qc-step-not-completed' and 'qc-step-completed' in `testClass` and remove `testClasses`.

Also what do you think about adding `dataTest..`  or `cypress` as a suffix to all the variables related to cypress? this way we have. aclear way of identifying everything related to integration testing across all of the ui

Reply:
Hmm.. good point. That would minimize the impact in the çomponent. I've renamed the component to `dataTestClass`. Where possible, I agree it'd be good to prefix these variables.